### ISSUE TITLE
[v1.4.0] Release Tracker

### ISSUE THREAD

**Issue by mingbowan** [pytorch/pytorch#30331]
The branch v1.4.0 has been cut.

If you need any particular patches onto this branch, please comment below and send a PR to v1.4.0 branch (instead of master).

---

[Current PRs open against the v1.4.0 branch](https://github.com/pytorch/pytorch/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+base%3Av1.4.0)

**Comment by gottbrath** [comment]
was trying to get https://github.com/pytorch/pytorch/pull/30288 "updates to quantization doc" under the wire. It should be low risk but it does change both doc rst files and pytorch docstrings. I currently have three failures in CircleCI .. two of them seem to be pre-existing false positives but one might be a flaky test. Will flag when I get this landed and also create a PR against the release branch.

**Comment by gottbrath** [comment]
ok .. the doc changes landed on master. The new PR on the v1.4.0 is posted and is #30372

**Comment by rohan-varma** [comment]
Request for patch: https://github.com/pytorch/pytorch/pull/30545
Status: merged into master at 5c6705e after the 1.4 branch release

**Comment by vishwakftw** [comment]
https://github.com/pytorch/pytorch/pull/30575 adds notes about ordering on singular values and eigenvalues output by torch.svd and torch.symeig respectively.

---
Status: PR open against v1.4.0 at #30575

**Comment by jamesr66a** [comment]
Request for patch: https://github.com/pytorch/pytorch/pull/30629

Backwards-compatibility fix for a change landed shortly before the branch cut

**Comment by yf225** [comment]
Request for patch: https://github.com/pytorch/pytorch/pull/30668

It fixes high-pri issues https://github.com/pytorch/pytorch/issues/30508 and https://github.com/pytorch/pytorch/issues/30462 which prevent users from doing any of the following:
1. Moving `torch::nn` modules with undefined parameters/buffers to a different device/dtype
2. Optimizing `torch::nn` modules with undefined parameters

**Comment by yf225** [comment]
Request for patch: https://github.com/pytorch/pytorch/pull/30684

It is a 3-line fix that removes `namespace F = torch::nn::functional` from `torch/nn/modules/batchhnorm.h`, so that people don't have to define `torch::nn::functional` as `F` if they don't want to. Fixes issue https://github.com/pytorch/pytorch/issues/30682.

**Comment by nairbv** [comment]
https://github.com/pytorch/pytorch/pull/30729 cherry-picks changes to torch.where that were made on the 1.3.1 branch but hadn't been ported to master at the time of the 1.4.0 branch cut.

**Comment by jlin27** [comment]
Request for patch: [#30648 ](https://github.com/pytorch/pytorch/pull/30648)

Update docs persons of interest to add back the MSFT folks for ONNX Governance

**Comment by iseeyuan** [comment]
Request for patch: #30747 

It's to export the operator names from Python API for custom build.

**Comment by Krovatkin** [comment]
https://github.com/pytorch/pytorch/pull/30750

This turns off the new profiling graph executor for 1.4

**Comment by dreiss** [comment]
#30776 Fixes an error message.

**Comment by rohan-varma** [comment]
Request for patch: https://github.com/pytorch/pytorch/pull/30752

Status: merged into master at 1350b99

**Comment by jlin27** [comment]
Request for patch: [#30757](https://github.com/pytorch/pytorch/pull/30757)

Restructure docs organization and naming

**Comment by leofang** [comment]
Request for patch: #24947 

This is a bug fix (for example, see cupy/cupy#2589) for interoperating with other Python GPU libraries like CuPy and Numba that support the `__cuda_array_interface__` protocol. This patch ensures the compliance of the protocol. 

cc: @awthomp @beam2d @jakirkham

**Comment by houseroad** [comment]
Request for patch: https://github.com/pytorch/pytorch/pull/30939

Support exporting aten::copy_ and aten::index_put to ONNX opset 11

**Comment by yf225** [comment]
Request for patch: #31166

This PR removes the `transposed` argument from `Conv{1,2,3}dOptions`, so that users are encouraged to use `ConvTranspose{1,2,3}d` instead if they want to do deconvolution.

**Comment by houseroad** [comment]
Request for patch: https://github.com/pytorch/pytorch/pull/31170

Add Support for ONNX Interpolate Add Scales Params

**Comment by zasdfgbnm** [comment]
Should we include this? https://github.com/pytorch/pytorch/pull/31279

**Comment by mingbowan** [comment]
We will start building v1.4 RC release. No more patches after this comment.

cc: @peterjc123

**Comment by driazati** [comment]
Is it possible to still include #31390?

**Comment by zhangguanheng66** [comment]
And this one https://github.com/pytorch/pytorch/pull/29001. This is a patch to avoid BC breaking. It would be great to include it in v1.4.0 release.

**Comment by mingbowan** [comment]
I'll let @gchanan  or @soumith  to make the call

**Comment by BowenBao** [comment]
Request for patch from MSFT: https://github.com/pytorch/pytorch/pull/31397

Add ONNX Export Support to floor_divide

This PR is required to unblock onnx export for torchvision models, fixing the breakage introduced recently from torchvision side.

cc @spandantiwari @lara-hdr @fmassa @houseroad

**Comment by eellison** [comment]
@mingbowan what can we do to land #30750 ? it is a critical patch.

**Comment by gottbrath** [comment]
What is the story on https://github.com/pytorch/pytorch/pull/30372 .. can we get this incorporated please?

**Comment by vishwakftw** [comment]
Is it possible to include #31432 as well? This fixes a segmentation fault in `unflatten` for named tensors.

**Comment by gchanan** [comment]
Folks -- we are trying to get the tests in a stable state before we start cherry-picking more PRs.  Hopefully that will be done tomorrow and we can resume.

Thanks!

**Comment by andreafonso** [comment]
Will 1.4.0 support Python 3.8?

**Comment by gchanan** [comment]
https://github.com/pytorch/pytorch/pull/31637

**Comment by ppwwyyxx** [comment]
Request for cherry-picking: https://github.com/pytorch/pytorch/pull/31716

**Comment by peterjc123** [comment]
Request for cherry-picking: #31816

**Comment by ngimel** [comment]
https://github.com/pytorch/pytorch/pull/31850 - a 1.4.0 cherry-pick of #28175

**Comment by lara-hdr** [comment]
Request for cherry-pick https://github.com/pytorch/pytorch/pull/31890 small PR for full ONNX support for Torchvision models (including KeypointRCNN).

cc @fmassa

**Comment by peterjc123** [comment]
Fix dll load logic for Python 3.8 on Windows: #32339

**Comment by soumith** [comment]
release is closed
